doc/generate_sample_repos.py

```diff
@@ -0,0 +1,92 @@
+"""This is a helper file to auto-generate the sample
+READMEs page for the mybinder documentation. It expects
+a couple of environment variables to be set corresponding
+to your github username / password, or to an access token
+you've created (see below for the proper variable names).
+
+The script grabs some metadata and the README text for
+all of the binder-examples repositories, and then constructs
+a markdown file that can be served on the mybinder.org
+documentation page.
+"""
+
+from github import Github
```

Comment:
How is this installed? I tried pypi and conda, and `github` was not found in either. Should we add it to the doc-requirements?